### ISSUE TITLE
Generate the three files hostname, public key and secret key

### ISSUE THREAD

**Issue by mriamnot** [cathugger/mkp224o#89]
Hi,

i am pretty newbie to command prompt and this, so please explain in layman terms.

i generated a onion address using command ^cute -y -n 1

it generated the host, public and secret key in command prompt but i want three files in the directory with the info.

How i can run a command to export it in form of three files hostname, public and secret key?

or any other way to do it. thanks.

**Comment by mriamnot** [comment]
i closed the cmd but copied the information, so will need to convert them manually, any info will be helpful.

**Comment by cathugger** [comment]
you need to use `-Y` command line option
```
  -Y [FILENAME [host.onion]]
                        parse YAML encoded input and extract key(s) to
                        filesystem
```
so if you have file named `x.txt` containing the keys in text form, you'd do something like `cat x.txt | ./mkp224o -Y` or `./mkp224o -Y x.txt`, and it would process all the keys it finds in there, printing hostname(s) and making directories for them.

**Comment by mriamnot** [comment]
> 

1 minute.

**Comment by cathugger** [comment]
probably can be closed
